### DISCUSSION TITLE
libceph: auth method 'x' error -13

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#11254] [Q&A]
The cephx disabled, but the libceph return error:

```
❯ ceph config get mon. auth_client_required
none

❯ ceph config get mon. auth_service_required
none

❯ ceph config get mon. auth_cluster_required
none
```

**Comment by travisn** [comment]
Why is cephx disabled? And what exactly is the question?

**Reply by zhucan** [reply]
my question is,  why the logs of message prints "libceph: auth method 'x' error -13 "?  and it happed because of that we don't disable cephx?

**Reply by travisn** [reply]
What log is this?

**Reply by zhucan** [reply]
/var/log/messages
